integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -543,8 +543,15 @@ metrics:
   # podAnnotations:
   #   prometheus.io/scrape: "true"
   #   prometheus.io/port: "19999"
+  #   prometheus.io/jobPort: 20002
+  #   prometheus.io/workerPort: 30000
   #   prometheus.io/path: "/metrics/prometheus/"
-  podAnnotations: {}
+  podAnnotations:
+    prometheus.io/path: /metrics/prometheus/
```

Comment:
Let's not add these podAnnotations as default values - the commented part is enough. The reason is the same as earlier that prometheus functionality should not be by default turned on, even though `metrics` is turned on, because there can be users using values in metrics but not using prometheus.

Reply:
Got it, thanks!